### PR TITLE
API adjustments for the HTOTI project

### PR DESCRIPTION
**Pull request by Eydcao**
1. an option to turn on/off voxelizer: as per the discussion between @yuanming-hu, @maajor and myself, we may need a de-coupled voxelizer (even a hda) in the future. The Houdini user may directly use the built-in pointsFromVDB node, so no reason for them to pay the overhead.
2. add a function to clear colliders (useful in cases involving moving walls, spheres etc) that is very common in Houdini's scene.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,4 @@ rendered
 particles_*
 sim_*
 *.bin
+.vscode/
```
Comment (yuanming-hu) [inline .gitignore:19]: Leave an EOL here?

**Review by yuanming-hu — APPROVED** [review]
LGTM!

**Comment by yuanming-hu** [comment]
Please feel free to merge after adding the EOL :-)

File: .gitignore (hunk 1 above)
Comment (Eydcao) [reply .gitignore]: done, but not sure why git-hub is 'hiding' the EOL
image

**Pull request merged**